cmd/homi/setup/flags.go

```diff
@@ -494,4 +494,10 @@ var (
 		Name:  "kip103-contract-address",
 		Usage: "kip103 contract address",
 	}
+
+	cancunCompatibleBlockNumberFlag = &cli.Int64Flag{
+		Name:  "cancun-compatible-blocknumber",
+		Usage: "cancunCompatible blockNumber",
+		Value: 0,
+	}
```

Comment:
This is my personal question I had before. If the Value of the developing hardfork Flag is zero, then private network created by homi will automatically activate cancunCompatible hardfork. Do you know if it is a intended behavior or not?

Reply:
I have no idea if it's intended or our devs followed the previous hardforks' conventions, like I did.

IMHO, it would be OK to leave it as it is because builders that are not familiar with Klaytn would have hard time configuring correct hardfork numbers on their own. Our docs and posts are mostly focused on the latest version and it might be confusing for them to find correct hardfork for their need and setup required hardforks (our hardforks have dependencies).

If we want to change that, I could be handled in a separate PR because it would require to change other hardfork values which may not be in the scope of this PR. However, if you have other opinions, I'm happy to discuss it with you in this conversation.

Reply:
Oh I mean that cancunCompatible hardfork is still on development, and I'm curious if the hardfork items on development can be activated in dev branch.

Reply:
Ah, that is another point.

Both options seems OK, if a user is compiling `homi` on the dev branch,
- probably interested in features under development, so enable those features by default
- probably familiar with Klaytn, so make the user configure hardfork number by his/her own

Given that both options are possible, I'd like to leave it as it is because
- we have been doing like that
- we don't have to add another PR for this when all features are developed and the hardfork is ready

Reply:
Great, thank you. It solves the curiosity.